itou/approvals/export.py

```diff
@@ -73,11 +49,25 @@ def export_approvals(export_format="file"):
     ws.title = "Export PASS SIAE " + current_dt.strftime(DATE_FMT)
 
     data = [FIELDS]
-
-    for approval in Approval.objects.valid():
-        new_line = _approval_line(approval)
-        if new_line:
-            data.append(_approval_line(approval))
+    approvals = Approval.objects.all().select_related("user").prefetch_related("jobapplication_set__to_siae")
+
+    for approval in approvals:
+        # The same approval can be used for multiple job applications.
```

Comment:
Puisque un numéro de PASS IAE peut être affecté à plusieurs candidatures, il faut itérer sur toutes les candidatures liées.